src/main/g8/build.sbt

```diff
@@ -12,7 +11,6 @@ lazy val root = (project in file(".")).
       "com.typesafe.akka" %% "akka-http"            % akkaHttpVersion,
       "com.typesafe.akka" %% "akka-http-spray-json" % akkaHttpVersion,
       "com.typesafe.akka" %% "akka-http-xml"        % akkaHttpVersion,
-      "com.typesafe.akka" %% "akka-stream"          % akkaVersion,
```

Comment:
Better to not remove this one. It is explicitly set to specify that we want to use the a different Akka Version which is not the one bundled with Akka HTTP.